app/lib/efile/nc/d400_calculator.rb

```diff
@@ -139,21 +139,39 @@ def calculate_line_19
       end
 
       def calculate_line_20a
-        @intake.state_file_w2s.reduce(0) do |sum, w2|
+        sum = 0
+
+        @intake.state_file_w2s.each do |w2|
           if w2.employee_ssn == @intake.primary.ssn
-            sum += w2.state_income_tax_amount.to_i
+            sum += w2.state_income_tax_amount&.round
```

Comment:
i changed this to `&.round` -- the client can enter a decimal amount here which is persisted in the DB as a decimal number. Going to `round` instead of `to_i` since the helper text for that field also indicates it should round the value.